proton

```diff
@@ -469,11 +470,11 @@ elif sys.argv[1] == "waitforexitandrun":
 elif sys.argv[1] == "getcompatpath":
     #linux -> windows path
     path = subprocess.check_output([wine_path, "winepath", "-w", sys.argv[2]], env=env, stderr=lfile)
-    sys.stdout.write(path.decode('utf-8'))
```

Comment:
Please note that this was done to maximize compatibility with as few changes as necessary. @nanonyme expressed some objections to this change in https://github.com/ValveSoftware/Proton/pull/9#discussion_r211950495

The same change also happens four lines below.